avocado/core/test.py

```diff
@@ -148,14 +148,14 @@ def record_and_warn(*args, **kwargs):
         self.whiteboard = ''
 
         self.running = False
-        self.time_start = None
-        self.time_end = None
+        self.time_start = 0
```

Comment:
I agree. My motivation was the fact, that some SW might not work properly with time == -1. For example python says '1970-01-01 00:59:59' to that.

I'll update all occurrences and we can change it to 0 if we get some bug requests.